base_img/init

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
There's a mix of `/bin/sh` and `/bin/bash` used in the scripts in this PR.. is that intentional? can we pick one or the other, just for consistency sake?

Reply:
Switched to `/bin/bash` in all cases except for the base_img scripts, which are running in the busybox context, and so don't have access to bash.